src/workerd/io/actor-sqlite.c++

```diff
@@ -402,6 +556,10 @@ kj::Maybe<kj::Promise<void>> ActorSqlite::onNoPendingFlush() {
 
 const ActorSqlite::Hooks ActorSqlite::Hooks::DEFAULT = ActorSqlite::Hooks{};
 
+kj::Promise<void> ActorSqlite::Hooks::scheduleRun(kj::Maybe<kj::Date> newAlarmTime) {
+  return kj::READY_NOW;
```

Comment:
Default implementation should probably throw an exception saying alarms are not implemented?